opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -144,9 +146,13 @@ class SpanKind(enum.Enum):
     CONSUMER = 4
 
 
-class Span:
+class Span(object):
```

Comment:
We are not Python2 compatible, pylint should issue an error here about useless object inheritance.

Reply:
Sorry, too much Python2 compatible code has made that almost a reflex response in me, removing. :sweat_smile: